superset/db_engine_specs/mssql.py

```diff
@@ -53,6 +53,7 @@ class MssqlEngineSpec(BaseEngineSpec):
     max_column_name_length = 128
     allows_cte_in_subquery = False
     allow_limit_clause = False
+    supports_multivalues_insert = True
```

Comment:
Please also add `supports_multivalues_insert` to `BaseEngineSpec`. I assume it will need to be disabled by default.

Reply:
Done in [217ad1f](https://github.com/apache/superset/pull/30222/commits/217ad1fb3378297b11ebe64862181d7424ea0677). Agreed with setting it to False by default: I expect many DB types support it, but since it's not currently defined in BaseEngineSpec at all we're not reducing any existing functionality with this, and setting it to True might break uploads for some DBs that don't support it.